src/hb-ot-name-table.hh

```diff
@@ -230,6 +228,7 @@ struct name
     TRACE_SANITIZE (this);
     return_trace (c->check_struct (this) &&
 		  likely (format == 0 || format == 1) &&
+		  likely (count <= 16382) &&
```

Comment:
Can you add a comment explaining how this number is derived and the reason for the check.